tests/views/layout.jade

```diff
@@ -15,6 +15,6 @@ html
         script(src='../bower/jasmine/lib/jasmine-core/jasmine-html.js')
         script(src='../bower/jasmine/lib/jasmine-core/boot.js')
         script(src='../bower/jasmine-jsreporter/jasmine-jsreporter.js')
-        script(src='../dist/db.min.js')
+        script(src='../dist/db.js')
```

Comment:
As I'm using Firefox, the console still doesn't utilize this, even though the Debugger does show the unminified source: https://bugzilla.mozilla.org/show_bug.cgi?id=670002

You want me to revert this?